.gitignore

```diff
@@ -29,7 +29,6 @@
 /opensearch-devops/
 /cross-cluster-replication/
 /opensearch-cli/
-/.github/
```

Comment:
Excluded because there's now a .github folder. It conflicts with this project but is safely ignored.